bootstrap/pkg/apis/apps/group.go

```diff
@@ -155,6 +155,14 @@ const (
 	EXISTING_ARRIKTO = "existing_arrikto"
 )
 
+// Plugin kind used starting from v1beta1
+const (
+	AWS_PLUGIN_KIND              = "KfAwsPlugin"
+	GCP_PLUGIN_KIND              = "KfGcpPlugin"
+	MINIKUBE_PLUGIN_KIND         = "KfMinikubePlugin"
+	EXISTING_ARRIKTO_PLUGIN_KIND = "KfExistingArriktoPlugin"
+)
```

Comment:
Hey Gabriel
Thanks for the this!
What about KfDefs for the kfctl_k8s? How would they be converted?

Reply:
this is only used for plugins.  from this file it doesn't have any plugins?
https://github.com/kubeflow/manifests/blob/master/kfdef/kfctl_k8s_istio.yaml